README.md

```diff
@@ -1,43 +1,51 @@
-# Foundation Bootstrap Team
+# The OpenJS Foundation Cross Project Council
 
-This is the operating repository for the Foundation Bootstrap Team.
+The OpenJS Foundation Cross Project Council (CPC) is the technical governing body of the OpenJS Foundation. It is described in the [CPC Charter](./CPC-CHARTER.md)
 
-The Foundation Bootstrap Team is the temporary body tasked with managing community recommendations for the transition of the merged Node.js and JS Foundation to a permanent governance model.
+## CPC Scope
 
-The files in this repository are living documents and will never represent finished governance. Once we reach consensus on governance, this repository will be archived and the documents will be moved to their new home in a Foundation specific Github org.
+**The CPC is responsible for ensuring collaboration is the driving principle within a Project, between OpenJS Foundation Projects, and between OpenJS Foundation Projects and the broader community.**
```

Comment:
I'll try to come up with something for this sentence, but if not, two things:

* Please remove the bold text? Pretty please?
* This is a run-on sentence and difficult to understand. It should be re-worded and/or split into more than one sentence.

Reply:
(Deleted my previous comments because I think they're wrong because I was misreading the sentence because yikes!)

Reply:
I think this sentence should be removed or else completely rewritten with a careful eye towards being obviously meaningful. As it stands, it is difficult to see anything meaningful in it. Let's break this sentence down:

> The CPC ensures collaboration is the driving principle within a Project.

*The* driving principle? I'm not even sure it's possible for the CPC to do that. What do they do? Check on the project? Or actively manage the project if "collaboration" is not "the driving principle"? How does one evaluate whether collaboration is "the driving principle" of anything?

> The CPC ensures collaboration is the driving principle between OpenJS Foundation Projects.

What does that even mean? What is "the driving principle between OpenJS Foundation Projects"? There aren't driving principles between entities. That's not a thing.

> The CPC ensures collaboration is the driving principle between OpenJS Foundation Projects and the broader community.

Same as above, but even more so.

Reply:
Maybe something more like this?

```suggestion
The CPC primary role is to foster an environment of collaboration. That environment extends within and between OpenJS Foundation Projects. It also extends between the Projects and the larger community.
```

*How* the CPC will do that is left as an exercise to the CPC itself, I suppose, but I think that sentence at least as meaning. For example, it is much easier for me to understand what a collaborative-environment-between-projects looks like than collaboration-as-driving-principle-between-projects.